azure/Guidelines.md

```diff
@@ -88,80 +86,64 @@ All Azure APIs **MUST** use explicit versioning. The Microsoft REST API guidelin
+
+There are three groups of changes that may happen to an API.
+
+1. Changes made to an **EXISTING** API version due to security or compliance reasons.  We shall refer to these types of changes as _Compliance changes_.
+2. Changes made to an API that may cause a client making the API call to fail, such as removal of an endpoint or property or changing the format of the body.  We refer to these types of changes as _Breaking changes_.
+3. Additive changes made to an API that do not cause a client making the API call to fail, such as the addition of a new optional property or a new endpoint.  We refer to these types of changes as _Evolutionary changes_.
+
+With the exception of _Compliance changes_ (which are extremely rare), Azure services **MUST** update the version number of their API whenever there is a change to the API, no matter how small.  Customers will "lock the API version" so that their code does not fail when the service introduces new features.  They rely on the fact that an API version is a contract with the services that will never change.
+
+A _breaking change_ is any change in the API that may cause client or service code making the API call to fail. Obvious examples of such a change are the removal of an endpoint, adding or removing a required field or changing the format of the body (from XML to JSON for example). Even though we recommend clients ignore new fields, there are many libraries and clients that fail when new fields are introduced. Removing an endpoint from an API is always a _breaking change_.  Adding a new endpoint is always an _evolutionary change_.  Changes to properties may be _evolutionary_ or _breaking_ depending on the type of change and whether the change is to an input parameter or output parameter:
+
+| Property change        | Input        | Output       |
+|========================|==============|==============|
+| Remove a property      | Breaking     | Breaking     |
+| Add optional property  | Evolutionary | Breaking     |
+| Add required property  | Breaking     | Breaking     |
+| Data type change       | Breaking     | Breaking     |
+| Format change          | Breaking     | Breaking     |
+| Integer widens         | Evolutionary | Breaking     |
+| Integer narrows        | Breaking     | Evolutionary |
+| Add new value to enum  | Evolutionary | Breaking     |
+| Remove value from enum | Breaking     | Breaking     |
+| Optional to required   | Breaking     | Breaking     |
+| Required to optional   | Evolutionary | Breaking     |
+
+Breaking changes require prior approval of the Azure REST API review board. In the case of deprecation, follow the API deprecation policy (below).  If the service is using SemVer for versioning, breaking changes constitute a major version change.
+
+Evolutionary changes do not require prior approval (but still need a version bump).  If the service is using SemVer for versioning, evolutionary changes constitute a minor version change.
```

Comment:
Is it possible to have a patch level version change or is `Major.Minor` the only format covered under the guidelines?

Reply:
Looking through the larger document, I don't see the `Major.Minor` explicitly defined, but that is expected.

Reply:
The current document states that only date-versioning is allowed.  However, some service groups have decided at a business level to adopt major.minor (e.g. cognitive services).  This is basically saying "don't use semver, but if you do - here are the rules"

Reply:
The current document recommends M.m versions and "group versions" which are date based

https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#121-versioning-formats